### ISSUE TITLE
Verification Error (Less deviation with glitches between reference plot and actual plot) of building latest models

### ISSUE THREAD

**Issue by deepak19015** [OpenModelica/OpenModelica#11153]
### Description

The output of the building latest models has many plots, each plot should overlap or have very less deviation with respect to reference plot. So, The models which has less difference with glitches or spikes between Reference plot and actual plot are listed below.

```
Buildings.Airflow.Multizone.Examples.OneOpenDoor
Buildings.Airflow.Multizone.Examples.TrickleVent
Buildings.BoundaryConditions.WeatherData.Validation.OverAYear_usingOneYearData
Buildings.Examples.ScalableBenchmarks.BuildingVAV.Examples.OneFloor_OneZone
Buildings.Experimental.DHC.Loads.BaseClasses.Examples.CouplingRCZ1Valve
Buildings.Fluid.Chillers.Validation.Carnot_TEva_reverseFlow
Buildings.Fluid.HeatExchangers.Examples.WaterHeater_T
Buildings.Fluid.HeatExchangers.Radiators.Examples.RadiatorEN442_2
Buildings.Fluid.HeatExchangers.Validation.ConstantEffectiveness
Buildings.Fluid.MixingVolumes.Examples.MixingVolumeMoistAir 
Buildings.Fluid.MixingVolumes.Validation.MixingVolume
Buildings.Obsolete.Fluid.Movers.Validation.ControlledFlowMachine
Buildings.Obsolete.Fluid.Movers.Validation.ControlledFlowMachineDynamic
Buildings.ThermalZones.Detailed.Validation.BESTEST.Cases6xx.Case685
Buildings.ThermalZones.Detailed.Validation.BESTEST.Cases6xx.Case695
```

### Steps to Reproduce

All the above listed model can be found on this link,
(Source: https://libraries.openmodelica.org/branches/master/Buildings_latest/Buildings_latest.html) after that go through those links which are on verified column. 

### Expected Behavior

The reference plot and actual plot should overlap.

### Screenshots

![spikes-deviation](https://github.com/OpenModelica/OpenModelica/assets/139556265/098d83aa-a950-44a5-8208-0a507e6e1b05)

All the above mentioned models has similar type of mismatch in the reference plot and actual plot.

### Version and OS

- OpenModelica Version: OpenModelica 1.22.0~dev-250-g48ea6c4

- OS: Ubuntu 20.04 focal

Versions of used Modelica libraries:
Modelica 4.0.0
Buildings master (10.0.0)

**Comment by bilderbuchi** [comment]
> The reference plot and actual plot should overlap.

I get the idea in principle, but could it sometimes be that the reference actually recorded problematic behaviour, and OM surfaces this problem? 

For example, in the screenshot you give, the deviation at t~2420 looks like a glitch in the reference, or a really really fast transient at this one point (is that likely to be in an example model?). 
Did you confirm that the deviating actual results that you report are in fact erroneous?

**Comment by casella** [comment]
> > The reference plot and actual plot should overlap.

@bilderbuchi this statement doesn't imply by any means that we should change the actual plot - it could also be the other way round 😄 

> I get the idea in principle, but could it sometimes be that the reference actually recorded problematic behaviour, and OM surfaces this problem?

Absolutely. I believe this is almost always the case. My suggestion to @mwetter is that we set a tighter solver tolerance, e.g. 1e-7 or 1e-8 for these problematic models, so that both the tool generating the reference result and OpenModelica will use a more precise numerical approximation. I believe most of these issues will be automatically resolved this way. I'm waiting for feedback from him on the topic.

**Comment by mwetter** [comment]
@casella : Tightening the tolerance is good for the problematic models, unless these are huge and stall the CI testing. (See also my email reply from Sep 5)

**Comment by casella** [comment]
> @casella : Tightening the tolerance is good for the problematic models, unless these are huge and stall the CI testing. 

Of course. I think we can increase the tolerance gradually and see what happens.

> (See also my email reply from Sep 5)

e-mail backlog is going to kill me 😅 

Thanks!
